index.d.ts

```diff
@@ -139,5 +139,6 @@ declare module '@appfolio/react-native-upload' {
     static getFileInfo(path: string): Promise<FileInfo>;
     static cancelUpload(uploadId: uploadId): Promise<boolean>;
     static canSuspendIfBackground(): Promise<void>;
+    static shouldLimitNetwork(limit: boolean): Promise<void>;
```

Comment:
I think these two can actually be synchronous as they don't wait on the return value of the `NativeModule` call?

Reply:
Done!